config.sample.yaml

```diff
@@ -39,3 +39,5 @@ logging:
 # password requirements
 password:
   min_length: 8
+# username requirements
+username_validation_regex: '' #regex that selected usernames must match - default is '[a-zA-Z_\-=\.\/0-9]+'
```

Comment:
i think it would be nice to make it `validation_regex` as an item of `username` similar to password, so other requirements can easily be added later and just for consistency.